styles/ShoppingList.css

```diff
@@ -1,108 +1,110 @@
 @import './MainMenu.css';
 
-*{
-    margin:0 auto;
```

Comment:
why you changed the styles from not your feautre?

Reply:
We discussed it with Wiktoria because there was problem with menu and I said I can fix it within my pr as because I was doing one anyway

Reply:
Okay thanks